scripts/generate_tarball.sh

```diff
@@ -64,8 +64,6 @@ create_symlink eosio-pp eosio-pp
 create_symlink cdt-init cdt-init
 create_symlink eosio-wasm2wast eosio-wasm2wast
 create_symlink eosio-wast2wasm eosio-wast2wasm
-create_symlink eosio-wasm2wast cdt-wasm2wast
-create_symlink eosio-wast2wasm cdt-wast2wasm
```

Comment:
Why are these two symlinks removed?

Reply:
yea, should not do this. probably an error in a reverted commit.